### ISSUE TITLE
Installation with cargo fails due to missing `fontdb` dependency

### ISSUE THREAD

**Issue by jangxx** [typst/typst#5588]
### Description

I'm trying to install typst by running `cargo install --locked typst-cli` like mentioned in the documentation, but it fails with the following error:

```
  Downloading crates ...                                                                                                                                                                                                                        
   Downloaded typst-cli v0.12.0                                                                                                                                                                                                                 
   Installing typst-cli v0.12.0                                                                                                                                                                                                                 
 error: failed to select a version for `typst-kit`.
     ... required by package `typst-cli v0.12.0`
 versions that meet the requirements `^0.12.0` (locked to 0.12.0) are: 0.12.0
 
 the package `typst-cli` depends on `typst-kit`, with features: `fontdb` but `typst-kit` does not have these features.
  It has an optional dependency with that name, but that dependency uses the "dep:" syntax in the features table, so it does not have an implicit feature with that name.
 
 
 failed to select a version for `typst-kit` which could resolve this conflict
```

This is happening both under alpine Linux and Debian bookwork, more specifically inside of the `node:18-alpine` and `node:18-bookworm` docker images. I'm trying to install typst inside of these images so that I can use the CLI from my Node.js app, but the installation fails with the error above.

The specific commands from my dockerfile are as follows:

```dockerfile
FROM node:18-alpine

RUN apk add --no-cache cargo
RUN cargo install --locked typst-cli
```

### Reproduction URL

_No response_

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by laurmaedje** [comment]
Could you post your full Dockerfile? I can't reproduce this outside of Docker and the excerpt you posted fails with OpenSSL errors.

**Comment by jangxx** [comment]
Looks like I had an outdated cached version of `node:18-alpine` on my system. Retrying the docker build command with `--pull` to pull the latest version of the base image also failed with an OpenSSL error, which I then fixed. It looks like this problem only occurs in older versions of alpine, so we can close this.

For reference, this is the full dockerfile I used for testing that worked in the end:

```dockerfile
FROM node:18-alpine

RUN apk add --no-cache openssl-dev pkgconfig cargo
RUN cargo install --locked typst-cli

CMD [ "node", "-V" ]
```
